### PR TITLE
test last language

### PR DESCRIPTION
**Pull request by JuliaChernakov**

### CODE REVIEW

File: src/test/java/qa_java_beginners/JuliaChernakovHW12Test.java [hunk 1]
```diff
@@ -25,5 +25,23 @@ public void testPageDescription() {
...
+        getDriver().get("http://www.99-bottles-of-beer.net");
+
+        getDriver().findElement(By.xpath("//li/a[@href='/abc.html']")).click();
+        getDriver().findElement(By.xpath("//a[@href='m.html']")).click();
+        WebElement text = getDriver().findElement(By.xpath("//tr[last()]/td/a"));
+        String actualResult = text.getText();
+
+        Assert.assertEquals(actualResult, expectedResult);
+    }
 }
```
Comment (anonymoustuna) [inline src/test/java/qa_java_beginners/JuliaChernakovHW12Test.java:47]: Empty line.

**Review by anonymoustuna — APPROVED** [review]

File: src/test/java/qa_java_beginners/JuliaChernakovHW12Test.java (hunk 1 above)
Comment (JuliaChernakov) [reply src/test/java/qa_java_beginners/JuliaChernakovHW12Test.java:47]: Thank you for approval, I'll remove it in next commit.

Comment (zhunat) [reply src/test/java/qa_java_beginners/JuliaChernakovHW12Test.java:47]: Looks perfect to me

**Review by zhunat — APPROVED** [review]
your code looks good

**Pull request merged**